xdsl/backend/csl/print_csl.py

```diff
@@ -447,6 +447,8 @@ def attribute_value_to_str(self, attr: Attribute) -> str:
                 return str(val.data)
             case StringAttr() as s:
                 return f'"{s.data}"'
+            case DenseIntOrFPElementsAttr(data=ArrayAttr(data=data), type=typ):
+                return f"{self.mlir_type_to_csl_type(typ)}{{{", ".join(str(d.value.data) for d in data)}}}"
```

Comment:
Should probably be `attribute_value_to_str(d.value)` instead of `str(d.value.data)`?